release-1.6/release-notes-draft.md

```diff
@@ -0,0 +1,753 @@
+  using will work with Kubernetes 1.6.
+- The kube-discovery Deployment isn't used anymore when creating a kubeadm
+  cluster and shouldn't be used anywhere else either due to its insecure nature.
+- The Certificates API has graduated from alpha to beta. This is a
+  backwards-incompatible change since the alpha support is dropped, and
+  therefore kubeadm v1.5 and v1.6 don't work together (for example kubeadm v1.5
+  can't join a kubeadm v1.6 cluster).
+- Supporting only etcd3, with 3.0.14 as the minimum version.
+- `kubeadm reset` will no longer drain nodes automatically.  This is because the
+  credentials on nodes do not have permission to perform this operation.  We
+  have documented an [alternate
+  procedure](https://kubernetes.io/docs/getting-started-guides/kubeadm/#tear-down),
+  driven from the API server/master.
+
+## Cluster Ops
+### New Features
+### Notable Changes
+### Breaking Changes
+
+## Contributor Experience
+### New Features
+### Notable Changes
+### Breaking Changes
+
+## Docs
+### New Features
+### Notable Changes
+### Breaking Changes
+
+## Federation
+### New Features
+### Notable Changes
+- `kubefed` has graduated to beta: supports hosting federation on on-prem clusters, automatically configures `kube-dns` in joining clusters and allows passing arguments to federation components.
+### Breaking Changes
+
+## Instrumentation
+### New Features
+### Notable Changes
+### Breaking Changes
+
+## Network
+### New Features
+* Adds support for configurable DNS stub domains and upstream nameservers to kube-dns.
+  The following configuration options have been added to the `kube-system:kube-dns` ConfigMap:
+  ```
+  "stubDomains": {
+    "acme.local": ["1.2.3.4"]
+  },
+  ```
+  is a map of domain to list of nameservers for the domain. This is used
+  to inject private DNS domains into the kube-dns namespace. In the above
+  example, any DNS requests for *.acme.local will be served by the
+  nameserver 1.2.3.4.
+  ```
+  "upstreamNameservers": ["8.8.8.8", "8.8.4.4"]
+  ```
+  is a list of upstreamNameservers to use, overriding the configuration
+  specified in /etc/resolv.conf.
+
+### Notable Changes
+* An empty `kube-system:kube-dns` ConfigMap will be created for the cluster if one did not already exist.
+
+### Breaking Changes
+
+## Node
+### New Features
+### Notable Changes
+- The Docker-CRI implementation is Beta and is enabled by default in kubelet.  You can disable it by --enable-cri=false. See [notes on the new implementation]( https://github.com/kubernetes/community/blob/master/contributors/devel/container-runtime-interface.md#kubernetes-v16-release-docker-cri-integration-beta) for more details.
+- kubelet launches pods in a new cgroup hierarchy to better enforce quality of service.  Operators must drain all pods from their nodes prior to upgrade of the kubelet.  They must ensure the configured cgroup driver matches their associated container runtime cgroup driver.
```

Comment:
One more item to add - 
```
Hard eviction thresholds will be subtracted from the Allocatable capacity on nodes to improve node reliability. This can break existing clusters since the overall schedulable capacity would reduce after upgrading to v1.6. 
You can opt-out of this feature by specifying --experimental-allocatable-ignore-eviction=true. 
```